### PR TITLE
added react router and created folder structure.

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Added react-router to project.
Created folder structure like componenet/header/Header.jsx and pages/homepage/Home.jsx.
Added fontawesome cdn, component library cdn and mockman CSS cdn in index.html
Not any visual change

Files to check:
    - index.html
    - index.js
    - app.js
    - folder structure

Added Header component:
files to check:
- Header.jsx
-Header.css

Capture
Capture

### CODE REVIEW

**Review by omkarpatke — APPROVED** [review]

File: src/pages/homepage/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
```
Comment (PuneetP16) [inline src/pages/homepage/Home.jsx]: Suggestion: You can avoid writing import React on individual components as it is not necessary.

File: src/components/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { Header } from "./header/Header";
```
Comment (PuneetP16) [inline src/components/index.js:1]: Suggestion: you could export footer as you have exported Header component

**Review by PuneetP16 — APPROVED** [review]
So far folder structure looks good and suggested certain changes.

**Review by omkarpatke — APPROVED** [review]
suggestion- only look at why netlify deployment is failed.

File: src/components/header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+const Header = () => {
+	return (
+		<div className="header-container">
+			<header className="header">
+				<div className="header-sub-container">
+					<button className="btn btn-primary btn-icon btn-menu">
+						<i className="fa fa-bars fa-lg" aria-hidden="true"></i>
+					</button>
+					<Link to="/home" className="btn btn-link btn-hide header-link">
+						<img className="img-round logo-container" src={logo} />
```
Comment (omkarpatke) [inline src/components/header/Header.jsx]: suggestion - in img tag you want to the alt tag I think that's whys netlify deployment failed.

**Review by omkarpatke — APPROVED** [review]

**Review by thenorthkun — APPROVED** [review]

File: src/components/header/Header.jsx (hunk 3 above)
Comment (nikhiljugale007) [reply src/components/header/Header.jsx]: Added alt for image.
Also i forgot to run npm run build, will push that in new commit.

**Pull request merged**